entries/e/engadget.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Engadget": {
+    "domain": "engadget.com",
+    "tfa": [
+      "email"
+    ],
+    "notes": "2FA via email is automatically set up as soon as you create your account.",
```

Comment:
I don't think this qualifies as an exception or restriction.
```suggestion
```

Reply:
So do I just leave it as is?

Reply:
Remove the note please.